Snakemake/rules/merge_samples_vcfs.smk

```diff
@@ -0,0 +1,99 @@
+#
+# Files must be at the vcf folder compressed and indexed
+#
+import os
+configfile: "../config/tsinfer.yaml"
+
+def list_full_paths(directory):
+    return [os.path.join(directory, file) for file in os.listdir(directory)]
+
+rule all:
+    input:
+        expand(contig['vcfDir'] + "Chr{chromosome}.vcf.gz", chromosome = range(1, config['noChromosomes'] + 1))
+
+workdir: config['workdir']
+vcfdir = config['vcfDir']
+
+VCFs =[x for x in list_full_paths(vcfdir) if x.endswith(".vcf.gz")]
+
+# creates a list of all files for that chromosome
+files_chr = [name in VCFs if '{chromosome}' in name]
```

Comment:
I think here you'll need [name for name in VCF ... ]

Reply:
Yes. This whole bit does not really work - I'm troubleshooting